python/tools/create_inbox_message/create_inbox_message.py

```diff
@@ -0,0 +1,33 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+
+import ftrack_api
+
+session = ftrack_api.Session()
+
+recipients_user_names = ['<username>']
+message_content = 'Message body'
+
+# Query author and recipients. Note that the message will not appear if the
+# author and recipient is the same user.
+author = session.query(f"User where username is {session.api_user}").one()
+recipients = session.query(
+    "User where username in ('{}')".format("','".join(recipients_user_names))
+).all()
+
+# Create note / "internal email"
+note = session.create('Note', {
+    'author': author,
+    'content': message_content
+})
+
+# Add recipients
+for user in recipients:
+    recipient = session.create('Recipient', {
+        'note_id': note['id'],
+        'resource_id': user['id']
+    })
+    note['recipients'].append(recipient)
+
+# Persist changes to database / send "internal email"
+session.commit()
```

Comment:
please add a way to run as standalone (windows) using the : if __name__ == '__main__' as for other recipes:
https://github.com/ftrackhq/ftrack-recipes/blob/main/python/actions/create-report-0.0.0/hook/create_report.py#L232

Reply:
Is this really needed for this recipe? This is a snippet that runs when you run the script so don't see the reason to use that line, and it's not something that needs a connection to the event hub or so. Is that line needed for Windows?

Reply:
Yeap, that's for windows....